.travis.yml

```diff
@@ -41,12 +41,14 @@ install:
   - export PATH="$HOME/miniconda/bin:$PATH"
   - conda env create -f environment.yml -n eal
   - source activate eal
+  - conda list
```

Comment:
Was this still needing to be in here...?

Reply:
No I'll remove it